src/Vimeo/Vimeo.php

```diff
@@ -33,7 +34,7 @@ class Vimeo
     const CLIENT_CREDENTIALS_TOKEN_ENDPOINT = '/oauth/authorize/client';
     const VERSIONS_ENDPOINT = '/versions';
     const VERSION_STRING = 'application/vnd.vimeo.*+json; version=3.4';
-    const USER_AGENT = 'vimeo.php 3.0.2; (http://developer.vimeo.com/api/docs)';
+    const USER_AGENT = 'vimeo.php 3.0.3; (http://developer.vimeo.com/api/docs)';
```

Comment:
@Sventour Should this be bumped up to `3.0.4`?

Reply:
yup, good eye!